symbology/create-symbol-styles-from-web-styles/src/main/java/com/esri/samples/create_symbol_styles_from_web_styles/CreateSymbolStylesFromWebStylesSample.java

```diff
@@ -32,6 +32,7 @@
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
 import javafx.scene.layout.*;
```

Comment:
These imports should be expanded out - not anything you've done but we should update it. Use your import formatter on this file, or shout if this doesn't make sense.